test/TestCases/loaders/i18nTypeScriptModules/Readme.md

```diff
@@ -0,0 +1,16 @@
+# What is the test about?
+typescript generates with the settings AMD/es2015 module defines having two
+parameters, require and exports. The exports weill be filled with the exported
+data.
+
+This needs to be addressed by dojo-webpack-plugin.
+
+# How to use the *.ts typescript files?
+Install Typescript with its tsc transpiler somewhere.
+
+Go to this folder.
+
+tsc
+
+Edit the generated nls/* files and adapt for esLint with a line comment:
```

Comment:
Or just add a .eslintrc file the this directory

Reply:
You are right, I'll put a .eslintignore